### DISCUSSION TITLE
how customize error 500 page

### DISCUSSION THREAD

**Discussion by TrungRueta** [nuxt/nuxt#17955] [Questions]
Hi, i'm working new project use Nuxt v3 and need to customize static error pages ( 404, 400, 500 ... ).

with error 404 i found that we can create new `/pages/404.vue` and framework auto take it as template. But i can not do same way with error 500. Look like framework fixed error 500 template somehow.

does anyone found way to do it?

Thanks!

**Comment by eMeRiKa13** [comment]
Curions about the answer. How to customize the error 500 page.

Thank you

**Comment by lukaszflorczak** [comment]
There is a similar problem if I try to redirect the user to the `404` page manually. For example:
* User visits `/my/nonexistent/static/page` – it works and user is redirected to my custom 404 page.
* User visits `/page/with-async-data/:id` – page with data fetched from backend where I want to redirect user manually, based on error from `useAsyncData` => user is redirected to 404 page, but the default one, from Nuxt.
